scripts/typescript-compatibility-build.yml

```diff
@@ -65,7 +69,7 @@ jobs:
     pool:
       vmImage: $(VmImage)
     strategy:
-      maxParallel: 2
+      maxParallel: 1
```

Comment:
Is this necessary with the introduction of uploading version #s? If not, can we revert to `2`?

Reply:
On running with `maxParallel` set to 2, not all version numbers get written to the `compatMap.txt` artifact file on each iteration of the matrix. Usually ends up with alternate versions numbers getting written. since two at a time get picked up. Can iterate on an alternate approach for this (_maybe append to a variable_.) I'd be happy to do this work in a subsequent PR.

Reply:
Ok, sounds good